python/lancedb/__init__.py

```diff
@@ -69,3 +69,6 @@ def connect(
             raise ValueError(f"api_key is required to connected LanceDB cloud: {uri}")
         return RemoteDBConnection(uri, api_key, region, host_override)
     return LanceDBConnection(uri)
+
+
+EVENTS = events.Events()
```

Comment:
Should this be wrapped in a try/except? The constructors has a bunch of initialization code. Not sure what's the probability of an exception occurring there?

Reply:
I think exceptions should not occur here because more of the conditions in the constructor are simply calling os.environ.get() which should be exception safe.

Reply:
This seems exception safe

Reply:
main concern is just that this is in the critical path for `import lancedb` so any error means the user will not be able to use lancedb at all

Reply:
SG!